truewiki/views/edit.py

```diff
@@ -33,16 +33,20 @@ def save(user, old_page: str, new_page: str, content: str, payload) -> web.Respo
         body = source.create_body(wiki_page, user, "Edit", new_page=new_page, page_error=page_error)
         return web.Response(body=body, content_type="text/html")
 
+    # Truncate long edit messages
+    # The html textfield already limits this, but if someone wanted to they could modify the html
+    summary = (summary[:500] + "..(truncated)") if len(summary) > 500 else summary
```

Comment:
I am personally a fan of just retuning 400s when users try to be sneaky. This means we could in `web_routes.py` just check the length and return a 400 if it exceeds 500. What do you think?

Codewise, instead of a ternary we can also do:

```
if len(summary) > 500:
  summary = summary[:500] + "..(truncated)"
```

Reply:
Yeah, sort of a slap on the hand for trying to test the system?

Reply:
Exactly :)